docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+At this point, you can block the default agent ingress address at `connect.ngrok-agent.com:443`, which all agents use by default to connect outbound to ngrok’s network. This address resolves to a dynamic set of IP addresses, and blocking it at your network prevents any usage outside of this site-to-site connectivity use case in partnership with your vendor, such as developers utilizing ngrok to tunnel local development environments to the public internet.
+
+You should also block the ngrok-managed public domains:
+
+- ngrok.app
+- ngrok.dev
+- ngrok-free.app
+- ngrok-free.dev
+- ngrok.io
+- ngrok.pizza
+- ngrok.pro
+
+### How can we mitigate the risk of abuse?
+
+ngrok has a multi-pronged strategy for combating malicious use of our network, including automated systems that flag suspicious activity. We also disincentivize abuse through product restrictions on free and unverified ngrok accounts.
+
+Work with your vendor to design an architecture that [prevents unauthorized use](#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?) and uses the appropriate [encryption model](#how-does-ngrok’s-traffic-encryption-work?) between your services.
+
+See our [abuse page](https://ngrok.com/abuse) for details.
+
+## Privacy and compliance
+
+### What compliance certifications does ngrok maintain?
+
+ngrok complies with SOC 2, GDPR, EU-US DPF, and CCPA.
+
+You can request our SOC 2 report and view other documents about ngrok’s security measures, like annual penetration testing, on our [Trust Center](https://trust.ngrok.com/).
+
+### What data does ngrok have access to, and how long is it stored?
+
+ngrok’s access to your data depends on which [encryption model](#how-does-ngrok’s-traffic-encryption-work?) specified by your site-to-site connectivity architecture—reach out to your vendor for more details.
+
+In all cases, ngrok stores information about the machine where you run your agent, such as its IP address, operating system, CPU architecture, and anonymized details about the environment.
+
+ngrok also logs changes to account configurations, such as agent start/stop, API key creation, and more. You can see the list in our [Event Store](/docs/obs/reference/) documentation.
+
+We retain data about your site-to-site connection at the direction of your vendor, who can also delete data at any time.
+
+Read our [primer on data at ngrok](https://ngrok.com/blog-post/data-at-ngrok) for more details.
+
+### What security practices does ngrok follow?
+
+ngrok uses the shared security responsibility model, where we are responsible for the security of our network and deliver features you can configure to secure your services. You and your vendor are responsible for securing your site-to-site connectivity architecture.
```

Comment:
ngrok uses the shared security responsibility model where we are responsible for the security of our network, and we deliver features you can configure to secure your services. You and your vendor are responsible for securing your site-to-site connectivity architecture.

*Removed the first comma and added the word "we" before "deliver features..."